janitor/functions/sort_column_value_order.py

```diff
@@ -1,44 +1,47 @@
+"""Implementation of the `sort_column_value_order` function."""
 import pandas_flavor as pf
 import pandas as pd
+from janitor.utils import check_column
+from .remove_columns import remove_columns  # noqa: F401
 
 
 @pf.register_dataframe_method
 def sort_column_value_order(
-    df: pd.DataFrame, column: str, column_value_order: dict, columns=None
+    df: pd.DataFrame, column: str, column_value_order: dict, columns=[]
```

Comment:
Ah, no, this is probably a bad idea. (see [here](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments)).

In general, the safer pattern is to initialize with `columns=None` and `if columns is None: columns = []` in the function body. This ensures that `columns`, if not provided, is always the empty list every function call.

In this case, I think we can probably leave it as it was originally (separate if-else blocks), or have something like:
```python
columns = [] if columns is None else columns
```
and do what you did.

Reply:
Good call! I'm going to use the latter.

Reply:
Great catch! Made the change.